### ISSUE TITLE
Extra new line added after line that longer than 80 symbols

### ISSUE THREAD

**Issue by IvanGoncharov** [nodeca/js-yaml#222]
Test:
```js
var YAML = require('js-yaml');

data = "Exactly 81 symbols in first line. Let's cound finale ones:             9876543210\n test"
console.log(data);
console.log('===============================================================');
data = YAML.safeDump(data);
console.log(data);
console.log('===============================================================');
data = YAML.safeLoad(data);
console.log(data);
console.log('===============================================================');
```
Output:
```
$ node test.js 
Exactly 81 symbols in first line. Let's cound finale ones:             9876543210
 test
===============================================================
>-
Exactly 81 symbols in first line. Let's cound finale
ones:             9876543210

 test

===============================================================
Exactly 81 symbols in first line. Let's cound finale ones:             9876543210

 test

===============================================================
```
So lib add extra newline after string with 80+ symbols.

**Comment by streamnsight** [comment]
set your option for lineWidth to be more than 80 chars.
80 is the default

**Comment by IvanGoncharov** [comment]
@streamnsight Yes I know. It was one of the reason why I added `lineWidth` option.
But still this is workaround and not a solution.

**Comment by JKillian** [comment]
:+1: This behavior surprised me as well.

**Comment by dervus** [comment]
This should be now fixed by #274
